routes/tags.routes.js

```diff
@@ -1,9 +1,47 @@
 /* 
+  Todo
+  
   tags listing
   default sort - rank desc
 
   - searchQuery
   - pagination(bool) = false
   - pageSize
   - pageNumber
-*/
\ No newline at end of file
+*/
+
+import { Router } from 'express';
+const {
+  getAllTags,
+  getTagInfo,
+  createTag,
+  updateTag,
+  deleteTag,
+} = require('../controllers/tags/tags.controller');
+
+const router = Router();
+
+/**
+ * GET / - Get all the tags
+ */
+router.get('/', getAllTags);
+
+/**
+ * POST / - Create a new tag
+ */
+router.post('/', createTag);
+
+/**
+ * Get /:id - Get Infor regarding a speicific tag
+ */
+router.get('/:id', getTagInfo);
```

Comment:
getAllTags can have this optional :id param to get info for specific tag id.
This way we don't need 2 routes/controllers.